Examples/Website/templates/hockey.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
```

Comment:
Same as above, but putting here as well that we can't have the website included as-is.